extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -248,6 +248,12 @@ define('ext.wikia.adEngine.adContext', [
 
 		enableAdMixExperiment(context);
 
+		if (geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneCountries) ||
```

Comment:
Is it needed? We can set `context.opts` flag no matter if Fastlane is enabled (either prebid or old integration)